src/t_stream.c

```diff
@@ -1377,7 +1375,7 @@ int streamIDEqZero(streamID *id) {
  *
  * NOTE: this assumes that the caller had verified that 'start' is less than
  * 's->last_id'. */
-int streamIsContiguousRange(stream *s, streamID *start) {
+int streamRangeContainsTombstone(stream *s, streamID *start) {
```

Comment:
Should the return value be:
0: no tombstone.
1: contains tombstone.

Now it's opposite.